### PR TITLE
Added faculty only search

### PR DESCRIPTION
**Pull request by awiss**

### CODE REVIEW

File: server/api.py [hunk 1]
```diff
@@ -88,9 +82,12 @@ def detail_search():
     else:
         params = [{'last_name':name},{'first_name':name}]
 
+    print params
```
Comment (adelq) [inline server/api.py]: Can you get rid of this print statement?

File: server/api.py [hunk 2]
```diff
@@ -88,9 +82,12 @@ def detail_search():
...
     ids = set()
     final = []
     for param in params:
+      param['affiliation'] = 'FAC'
+    for param in params:
```
Comment (adelq) [inline server/api.py]: Is this a mistake? I think you duplicated this line and indented line 89 too far.

Comment (awiss) [inline server/api.py]: No this is right. It updates each of an array of parameter dictionaries.

File: server/api.py (hunk 1 above)
Comment (awiss) [inline server/api.py]: I fix it in the next PR merge plox

File: server/api.py (hunk 2 above)
Comment (adelq) [inline server/api.py]: Ah OK

File: server/api.py (hunk 1 above)
Comment (adelq) [inline server/api.py]: Cool

**Pull request merged**